Ships as Raw.txt

```diff
@@ -70,10 +70,10 @@ Speed 1|2
 Speed 2|2 1 
 Speed 3|1 1 1
 Speed 4|1 0 1 1 
-Front Arc 2 |RRB
-Side Arc 2 |RR
-Rear Arc 1 |RB
-Anti Squadron|K
+FrontArc 2 |RRB
+SideArc 2 |RR
+RearArc 1 |RB
+AntiSquadron|K

```

Comment:
I like the CamelCaseing here instead of 2 words.  Let's carry that elsewhere so that all the keys are 1 word, ie like lines 43-45.